### DISCUSSION TITLE
teveel onverwerkte telegrammen

### DISCUSSION THREAD

**Discussion by ronniebee** [dsmrreader/dsmr-reader#1483] [Questions & answers / How-to's] [answered]
Ben al uren bezig mijn reader weer aan de praat te krijgen. 
configuratie""
Docker
>home assistant 

Heb de usb poort aangepast ( was veranderd van usb1 naar usb0)
draai de laatste versie van xirzix (docker compose)

wie kan mij adviseren?
```
DSMR-READER
    App / Python / Database                                                       v4.13 / v3.9.2 / postgresql
    BE sleep / DL sleep / Retention / Override                                     5.0s / 5.0s / 672h / False
    Latest telegram version read / Parser settings                                                 "50" / "4"

DATA
    Telegrams total (est.)                                                                              25631
    Consumption records electricity / gas (est.)                                                   1396 / 280

UNRESOLVED ISSUES
    Te veel onverwerkte telegrammen: 11703                                                     19 uur geleden
    Dagstatistieken lopen achter                                                        1 dag, 14 uur geleden

```

**Comment by dennissiemensma** [comment]
Bedankt voor je melding. Ik denk dat je het beste kunt beginnen door in de logfiles te kijken. Zie je daar toevallig foutmeldingen?

**Comment by ronniebee** [comment]
Hoi Dennis, ik kan deze files niet benaderen.  als ik :

docker$ sudo supervisorctl status
[sudo] wachtwoord voor XXXX: 
sudo: supervisorctl: opdracht niet gevonden

Ben geen expert met terminal :-)
of:
@Homeassistant:  sudo tail -f /var/log/supervisor/dsmr_datalogger*
tail: kan '/var/log/supervisor/dsmr_datalogger*' niet openen om te lezen: Bestand of map bestaat niet
tail: geen bestanden meer over
@Homeassistant:

**Comment by dennissiemensma** [comment]
Ik kan je op Docker-gebied niet echt helpen. Je kunt het beste zelf even oude issues zoeken voor logs, wellicht werkt dit voor je: xirixiz/dsmr-reader-docker/issues/158

**Comment by dennissiemensma** [comment, marked as answer]
Bleek veroorzaakt door #1282
